plugins/woocommerce/includes/class-wc-ajax.php

```
@@ -4249,6 +4250,230 @@ private static function render_variation_html( WC_Product $product_object, WC_Pr
+
+		if ( ! is_array( $raw_reviews ) || empty( $raw_reviews ) ) {
+			wp_send_json_error(
+				array( 'message' => __( 'No reviews submitted.', 'woocommerce' ) ),
+				400
+			);
+		}
+
+		$author_name  = $order->get_billing_first_name() . ' ' . $order->get_billing_last_name();
+		$author_email = $order->get_billing_email();
+		$user_id      = $order->get_customer_id();
+		$moderate     = get_option( 'comment_moderation' );
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$moderate'. (undefined)

(UnusedLocalVariable)